lib/bot.js

```diff
@@ -11,6 +11,10 @@ const NICK_COLORS = ['light_blue', 'dark_blue', 'light_red', 'dark_red', 'light_
   'dark_green', 'magenta', 'light_magenta', 'orange', 'yellow', 'cyan', 'light_cyan'];
 const patternMatch = /{\$(.+?)}/g;
 
+function caseInsensitiveExists(item, itemList) {
+  return itemList.some(i => i === item.toLowerCase());
```

Comment:
think you might as well inline this in `ignoredIrcUser` and `ignoredDiscordUser` - the extra layer of indirection probably just makes it harder to read

Reply:
Fixed.